webbpsf/opds.py

```diff
@@ -1084,6 +1087,15 @@ def __init__(self, name='Unnamed OPD', opd=None, opd_index=0, transmission=None,
         v2v3 : tuple of 2 astropy.Quantities
             Tuple giving V2,v3 coordinates as quantities, typically in arcminutes, or None to default to
             the master chief ray location between the two NIRCam modules.
+        control_point_instr: str
+            Name of the instrument where the OTE control point is located. Default: 'nircam'.
+            The OTE control point is the field point to which the OTE has been aligned and defines the field angles
+            for the field-dependent SM pose aberrations.
+        control_point_fieldpoint: str
+            Name of the field point where the OTE control point is located, on instrument defined by "control_point_instr". 
+            Default: 'nrca3_full'.
+            The OTE control point is the field point to which the OTE has been aligned and defines the field angles
+            for the field-dependent SM pose aberrations.
```

Comment:
Under what scenarios, if any, would we want to change these control point parameters? And can't the control point instrument be inferred from the control point field point name?

Reply:
I've wondered about whether NRCA3_FULL or NRCA3_FP1 should be the control point, but regardless, I favor having the flexibility to adjust it if we ever need it.

As for the "control_point_instrument", yes, we can infer it from the field point. I'll make the change.